app/routes/view-claim.js

```diff
@@ -127,6 +130,8 @@ module.exports = {
           })
         : [])
 
+      const status = { key: { text: 'Status' }, value: { html: `<span class='govuk-tag ${getStyleClassByStatus(formatStatusId(statusId))}'> ${upperFirstLetter(claimStatus?.status)} </span>` } }
+      const ClaimDate = { key: { text: 'Claim date' }, value: { html: convertDateToFormattedString(createdAt) } }
```

Comment:
Why is this capitalised? Should be "claimDate" not "ClaimDate"

Reply:
This reference is a claim reference from my test